### ISSUE TITLE
Add example applications 

### ISSUE THREAD

**Issue by AyushExel** [AyushExel/trolo#26]
There can be a table with a bunch of popular example applications of CV using trolo

**Comment by pks916** [comment]
@AyushExel please elaborate

**Comment by AyushExel** [comment]
Well there are a bunch of CV examples realted to vehicle counting, sports etc. We can create those examples using trolo and add them in the readme as a Table of examples

**Comment by pks916** [comment]
I will take it up

**Comment by hardikdava** [comment]
@AyushExel @pks916 We already have an utility function which converts our prediction to `supervision.Detection`. This enables whole supervision application.

https://supervision.roboflow.com/latest/cookbooks/

**Comment by AyushExel** [comment]
Yeah but we need them here too for visibility and integration

**Comment by hardikdava** [comment]
@AyushExel I am just guiding a path here. Not saying we should not do it.

**Comment by AyushExel** [comment]
Yup sounds good. @pks916 can you take a look at some of the supervision examples listed in the cookbook above? YOu can just use trolo models there

**Comment by pks916** [comment]
yes, saw them earlier. will do
